wp-includes/widgets/class-wp-widget-media-image.php

```diff
@@ -317,10 +317,6 @@ public function render_control_template_scripts() {
 						echo sprintf( __( 'Current image: %s' ), '{{ data.currentFilename }}' );
 					?></p>
 				<# } #>
-			<# } else { #>
-				<div class="attachment-media-view">
-					<p class="placeholder"><?php echo esc_html( $this->l10n['no_media_selected'] ); ?></p>
```

Comment:
That's the idea yes. You'll see that this change brings the image widget in line with the video and audio widgets, where they too lack this `else` statement.